### DISCUSSION TITLE
博士论文中的学术评语和答辩委员会决议书怎么使用扫描的pdf文件呢

### DISCUSSION THREAD

**Discussion by GanjinZero** [tuna/thuthesis#803] [Q&A] [answered]
example提供的代码只能用tex文件

**Comment by zggl** [comment]
\usepackage{pdfpages}
\includepdf[pages={1-480}]{IntroductiontoRoboticsACASaeedBNiku.pdf}

On Fri, Jun 24, 2022 at 12:19 PM Zheng Yuan ***@***.***>
wrote:

> example提供的代码只能用tex文件
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/tuna/thuthesis/discussions/803>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD66RNL5TVQBRBTQEXLFQF3VQUZMDANCNFSM5ZWOQOLQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
      祝好！

赵国亮
========================================================

通信地址：1. 呼和浩特 大学西路235号 内蒙古大学 卓越楼913
               PHONE:18604604983

E-Mail ： ***@***.******@***.***
========================================================

**Comment by GanjinZero** [comment, marked as answer]
```
\begin{comments}
% \begin{comments}[name = {指导小组学术评语}]
% \begin{comments}[name = {Comments from Thesis Supervisor}]
% \begin{comments}[name = {Comments from Thesis Supervision Committee}]

%   论文提出了……
\label{page:comments}
\addtocontents{toc}{\protect \contentsline {chapter}{指导教师学术评语}{\pageref{page:comments}}{} }
\includepdf[pages={1-2}]{data/comment.pdf}

\end{comments}
```
